geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/ListGatewaysCommandDUnitTest.java

```diff
@@ -452,8 +452,10 @@ public void testListGatewaysReceiversOnlyFalseAndSendersOnlyFalseReturnsSendersA
 
   @Test
   public void testListGatewaysWithOneDispatcherThread() {
-    String command =
-        "create gateway-sender --id=ln_Serial --remote-distributed-system-id=2 --dispatcher-threads=1";
+    String command = CliStrings.CREATE_GATEWAYSENDER + " --" + CliStrings.CREATE_GATEWAYSENDER__ID
```

Comment:
If you use the `CommandStringBuilder` class with the `addOption()` method, you can avoid having to manually put in the dashes and equals signs, like so:

```
String command = new CommandStringBuilder(CREATE_GATEWAYSENDER)
        .addOption(CREATE_GATEWAYSENDER__ID, "ln_Serial")
        .addOption(CREATE_GATEWAYSENDER__REMOTEDISTRIBUTEDSYSTEMID, "2")
        .addOption(CREATE_GATEWAYSENDER__DISPATCHERTHREADS, "1")
        .getCommandString();
```

This helps make commands easier to read and reduces the chance of typos when writing them.

Reply:
Nice! I take a look on test above, so I do on this way, but this proposal is better.